frontend/src/pages/projects/screens/detail/overview/serverModels/PlatformSelectSection.tsx

```diff
@@ -1,36 +1,49 @@
 import * as React from 'react';
 import { Alert, Gallery, Stack, Text, TextContent } from '@patternfly/react-core';
 import CollapsibleSection from '~/concepts/design/CollapsibleSection';
+import { useAppContext } from '~/app/AppContext';
+import SelectNIMCard from './SelectNIMCard';
 import SelectSingleModelCard from './SelectSingleModelCard';
 import SelectMultiModelCard from './SelectMultiModelCard';
 
-const PlatformSelectSection: React.FC = () => (
-  <CollapsibleSection title="Serve models" data-testid="section-model-server">
-    <Stack hasGutter>
-      <TextContent
-        data-testid="no-model-serving-platform-selected"
-        style={{ paddingLeft: 'var(--pf-v5-global--spacer--md)' }}
-      >
-        <Text component="small">
-          Select the type of model serving platform to be used when deploying models from this
-          project.
-        </Text>
-      </TextContent>
-      <Gallery
-        hasGutter
-        minWidths={{ default: '100%', lg: 'calc(50% - 1rem / 2)' }}
-        maxWidths={{ default: '100%', lg: 'calc(50% - 1rem / 2)' }}
-      >
-        <SelectSingleModelCard />
-        <SelectMultiModelCard />
-      </Gallery>
-      <Alert
-        isInline
-        variant="info"
-        title="The model serving type can be changed until the first model is deployed from this project. After that, if you want to use a different model serving type, you must create a new project."
-      />
-    </Stack>
-  </CollapsibleSection>
-);
+const PlatformSelectSection: React.FC = () => {
+  const { dashboardConfig } = useAppContext();
+  const { disableNIMModelServing } = dashboardConfig.spec.dashboardConfig;
```

Comment:
Use the hook please `useIsAreaAvailable` -- there is more to the setup than just a check of the dashboard config as a boolean state.

Reply:
As noted by Mriganka in [the other thread](https://github.com/opendatahub-io/odh-dashboard/pull/3156#discussion_r1767489008) -- we may want to make a custom hook that also does whatever the other check that is needed to see if we are enabled and good to go. I suggest maybe looking for the serving runtime on the cluster `nvidia-nim-runtime` -- but if you can think of another way, you could look into that.